frontend/src/modules/experimentRunLanes/actions.ts

```diff
@@ -47,12 +47,12 @@ export function setRunLaneValidationStatus(lane: LaneInfo, status: ValidationSta
 }
 
 export function setRunLaneValidationTime(lane: LaneInfo) {
-	return async (dispatch: AppDispatch, getState: () => RootState) => {
-    const latestValidationTime = lane.datasets.reduce<string | undefined>((latest, dataset) => {
+  return async (dispatch: AppDispatch, getState: () => RootState) => {
+    const validationTime = lane.datasets.reduce<string | undefined>((latest, dataset) => {
```

Comment:
why rename `latestValidationTime` to `validationTime`.

Reply:
LatestValidationTime gives the impression there is more than one ... this is the only value available ... multiple time. Keeping it simple is part of making it clear in that case.